includes/calendar-era.md

```diff
@@ -1,4 +1,3 @@
 
 > [!IMPORTANT]
->  A new era in the <xref:System.Globalization.JapaneseCalendar> and <xref:System.Globalization.JapaneseLunisolarCalendar> begins on May 1, 2019. This change affects all applications that use these calendars. See [Handling a new era in the Japanese calendar in .NET](https://devblogs.microsoft.com/dotnet/handling-a-new-era-in-the-japanese-calendar-in-net/) for more information and to determine whether your applications are affected. See [Prepare your application for the Japanese era change](/windows/uwp/design/globalizing/japanese-era-change) for information on testing your applications on Windows systems to ensure their readiness for the era change.
-
+>  The Reiwa era, a new era in the <xref:System.Globalization.JapaneseCalendar> and <xref:System.Globalization.JapaneseLunisolarCalendar>, begins on May 1, 2019. This change affects all applications that use these calendars. See [Handling a new era in the Japanese calendar in .NET](https://devblogs.microsoft.com/dotnet/handling-a-new-era-in-the-japanese-calendar-in-net/) for more information and to determine whether your applications are affected. See [Prepare your application for the Japanese era change](/windows/uwp/design/globalizing/japanese-era-change) for information on testing your applications on Windows systems to ensure their readiness for the era change. See [Working with eras](~/docs/standard/datetime/working-with-calendars.md#working-with-eras) for features in .NET that support calendard with multiple eras and for best practices when working with calendars that support multiple eras.
```

Comment:
>The Reiwa era, a new era in the [](start = 3, length = 31)

I would rephrase this part with something like: 


Japanese calendars expected to new era every while. For example, on May 1, 2019 introduced a new era Reiwa. 


then you can continue what you have wrote 


This change affects all applications that use these calendars....

Reply:
forgot to mention this will make us not changing this doc again (I guess)

---
In reply to: [271383644](https://github.com/dotnet/dotnet-api-docs/pull/2224#discussion_r271383644) [](ancestors = 271383644)